wgpu/Cargo.toml

```diff
@@ -84,6 +84,7 @@ replay = ["serde", "wgc/replay"]
 angle = ["wgc/angle"]
 webgl = ["wgc"]
 emscripten = ["webgl"]
+vulkan-portability = ["wgc/vulkan-portability"]
```

Comment:
are we checking this on macOS CI?

Reply:
Sorry, vulkan sdk on macOS doesn't have command-line installation method, and I don't know how to correctly config CI to check this feature

Reply:
I'm Learning CI today

Reply:
ok, no worries